code/app.js

```diff
@@ -0,0 +1,39 @@
+window.addEventListener('load', ()=> {
+    let long;
+    let lat;
+    let tempDescription = document.querySelector('.temp-description');
+    let tempDegree = document.querySelector('.temp-degree');
+    let locationTimezone = document.querySelector('.location-timezone');
+
+    if(navigator.geolocation){
+        navigator.geolocation.getCurrentPosition(position =>{
+            long = position.coords.longitude;
+            lat = position.coords.latitude;
+
+            const proxy = "https://cors-anywhere.herokuapp.com/";
+
+            const api = `${proxy}https://api.darksky.net/forecast/c48561e642e430bdaa676adbd2ace2e7/${lat},${long}`;
+            fetch(api)
+            .then(response =>{
+                return response.json();

```

Comment:
Nice use of promises :). I think you could even simply this more with brace-less implicit return: `.then(response => response.json())`